Misc/NEWS.d/next/Library/2024-10-03-17-13-22.gh-issue-124944.YyLAzf.rst

```diff
@@ -0,0 +1 @@
+Add :attr:`SO_ORIGINAL_DST` to the :mod:`socket` module.
```

Comment:
```suggestion
Add ``SO_ORIGINAL_DST`` to the :mod:`socket` module.
```

We don't have a documented attribute so we cannot use the `attr` role (this is only used for cross references). In particular, docs failures should not be ignored.